src/main/resources/graphql/schema.graphqls

```diff
@@ -11,6 +11,8 @@ type Category {
 }
 
 type Query {
+    user(username: String!, password: String!): User!
```

Comment:
we should call it passwordHash rather than password.
also, username isn't necessary